Vexed01/Vex-Cogs
# New cog: StatTrack

I am currently in the process of testing this cog. It should be pretty stable.

Note: do not install other cogs on this branch. They are outdated.

To test this, use these commands:

`[p]repo add vex-stattrack-dev https://github.com/Vexed01/Vex-Cogs dev/botstats`
`[p]cog install vex-stattrack-dev stattrack`
`[p]load stattrack`

When this PR is merged this branch will be deleted, so you should delete the branch and then reinstall the cog. Your data won't be affected.

`[p]cog uninstall stattrack`
`[p]repo delete vex-stattrack-dev`
`[p]repo update` (or `[p]cog update`)

Then add the cog from my normal repo.

## To-do

- [x] Use a different JSON format for more storage efficiently
- [x] Command to view storage/memory usage
- [x] Show how long data is for in graphs
- [x] Try again at dark mode
